roles/osquery/tasks/install-Debian.yml

```diff
@@ -0,0 +1,51 @@
+---
+- name: Wait for apt lock
+  ansible.builtin.shell: "while fuser /var/lib/dpkg/{{ item }} >/dev/null 2>&1; do sleep 5; done;"
+  loop:
+    - lock
+    - lock-frontend
+  changed_when: false
+
+- name: Install apt-transport-https package
+  become: true
+  ansible.builtin.apt:
+    name: apt-transport-https
```

Comment:
This is no longer needed for Jammy, should be only for Focal (not sure if even still needed there), and marked for removal.

Reply:
Let's remove this with one big PR and keep it like everywhere.